cron/workspacesLoad.js

```diff
@@ -0,0 +1,169 @@
+        'unit': 'Count',
+    },
+    'workspace_hosts_launching_count': {
+        'name': 'HostsLaunching',
+        'unit': 'Count',
+    },
+    'workspace_hosts_ready_count': {
+        'name': 'HostsReady',
+        'unit': 'Count',
+    },
+    'workspace_hosts_draining_count': {
+        'name': 'HostsDraining',
+        'unit': 'Count',
+    },
+    'workspace_hosts_unhealthy_count': {
+        'name': 'HostsUnhealthy',
+        'unit': 'Count',
+    },
+    'workspace_hosts_terminating_count': {
+        'name': 'HostsTerminating',
+        'unit': 'Count',
+    },
+    'workspace_hosts_stopped_count': {
+        'name': 'HostsStopped',
+        'unit': 'Count',
+    },
+    'workspace_hosts_longest_launching_sec': {
+        'name': 'MaxLaunchingHostAge',
+        'unit': 'Seconds',
+    },
+    'workspace_hosts_longest_ready_sec': {
+        'name': 'MaxRunningHostAge',
+        'unit': 'Seconds',
+    },
+    'workspace_hosts_longest_draining_sec': {
+        'name': 'MaxDrainingHostAge',
+        'unit': 'Seconds',
+    },
+    'workspace_hosts_longest_unhealthy_sec': {
+        'name': 'MaxUnhealthyHostAge',
+        'unit': 'Seconds',
+    },
+    'workspace_hosts_longest_terminating_sec': {
+        'name': 'MaxTerminatingHostAge',
+        'unit': 'Seconds',
+    },
+    'workspace_uninitialized_count': {
+        'name': 'WorkspacesUninitialized',
+        'unit': 'Count',
+    },
+    'workspace_stopped_count': {
+        'name': 'WorkspacesStopped',
+        'unit': 'Count',
+    },
+    'workspace_launching_count': {
+        'name': 'WorkspacesLaunching',
+        'unit': 'Count',
+    },
+    'workspace_running_count': {
+        'name': 'WorkspacesRunning',
+        'unit': 'Count',
+    },
+    'workspace_longest_launching_sec': {
+        'name': 'MaxLaunchingWorkspaceAge',
+        'unit': 'Seconds',
+    },
+    'workspace_longest_running_sec': {
+        'name': 'MaxRunningWorkspaceAge',
+        'unit': 'Seconds',
+    },
+};
+async function sendStatsToCloudwatch(stats) {
+    const cloudwatch = new AWS.CloudWatch(config.awsServiceGlobalOptions);
+    const dimensions = [{Name: 'By Server', Value: config.workspaceCloudWatchName}];
+    const cloudwatch_metricdata_limit = 20; /* AWS limits to 20 items within each list */
+    const entries = Object.entries(stats).filter(([key, _value]) => {
+        return (key !== 'timestamp_formatted');
+    });
+
+    for (let i = 0; i < entries.length; i+= cloudwatch_metricdata_limit) {
+        const data = entries.slice(i, i + cloudwatch_metricdata_limit).map(([key, value]) => {
+            if (!(key in cloudwatch_definitions)) {
+                throw new Error(`Unknown datapoint ${key}!`);
+            }
+            const def = cloudwatch_definitions[key];
+            return {
+                MetricName: def.name,
+                Dimensions: dimensions,
+                Timestamp: stats.timestamp_formatted,
+                Unit: def.unit,
+                Value: value,
```

Comment:
If you add `StorageResolution: 1` then I think you get hires data: https://github.com/PrairieLearn/PrairieLearn/blob/2843028d43de1a4f4197bd448e2632ca9cf06f8f/cron/externalGraderLoad.js#L49